docs/api/cases/cases-api-get-case.asciidoc

```diff
@@ -42,11 +42,11 @@ default space is used.
 
 === {api-examples-title}
 
-Returns case ID `a18b38a0-71b0-11ea-a0b2-c51ea50a58e2` without comments:
+Returns case ID `31cdada0-02c1-11ed-85f2-4f7c222ca2fa`:
```

Comment:
Nit - not edited here, but I noticed in the prerequisites it states `depending on the owner of the cases you’re seeking`, but as this is only retrieving a single case, should this be edited to `owner of the case...`?

Reply:
Good catch, thanks!